modules/react/select/stories/Select.stories.mdx

```diff
@@ -34,79 +34,145 @@ yarn add @workday/canvas-kit-react
 
-### Ref Forwarding
+1. Set the `nonInteractiveIds` prop of `Select` to an array of disabled item `id`s. This will
+   disable interaction for those items and exclude them from type-ahead.
 
-Select supports [ref forwarding](https://reactjs.org/docs/forwarding-refs.html). It will forward
-`ref` to its underlying `<select>` element.
+2. Set the `aria-disabled` attribute of all disabled `Select.Item`s to `true`. This ensures the
+   items are styled as disabled.
 
-<ExampleCodeBlock code={RefForwarding} />
+The following example assigns a `disabled` property to the disabled items in the `items` array.
+`disabled` is used to populate `nonInteractiveIds` and set `aria-disabled` for the disabled items.
 
-### Grow
+<ExampleCodeBlock code={DisabledOptions} />
 
-Set the `grow` prop of the wrapping Form Field to `true` to configure the Select to expand to the
-width of its container.
+### With Icons
 
-<ExampleCodeBlock code={Grow} />
+Use `Select.Item.Icon` to render an icon for a `Select.Item`. The `icon` prop for `Select.Item.Icon`
+accepts [system icons](/assets/system-icons/) from `@workday/canvas-system-icons-web`.
 
-The `grow` prop may also be applied directly to the Select if Form Field is not being used.
+In order to render the icon for the selected item in the `Select.Input`:
 
-### Label Position
+1. Obtain a reference to the `model` by registering your `items` with `useSelectModel`.
+2. Get the selected item:
+   `const selectedItem = model.navigation.getItem(model.state.selectedIds[0], model)`
+3. Pass the icon for the selected item to the input:
+   `<Select.Input inputStartIcon={selectedItem.value.icon}>`
 
-Set the `labelPosition` prop of the wrapping Form Field to designate the position of the label
-relative to the Select. `labelPosition` accepts the following values:
+<ExampleCodeBlock code={WithIcons} />
 
-- `FormField.LabelPosition.Top` (Default)
-- `FormField.LabelPosition.Left`
+Note that `Select.Input` will only render an icon if an item is selected.
 
-<ExampleCodeBlock code={LabelPosition} />
+### Grow
 
-### Required
+Set the `grow` prop of the wrapping `FormField` to `true` to configure the `Select.Input` to expand
+to the width of its container.
 
-Set the `required` prop of the wrapping Form Field to `true` to indicate that the field is required.
-Labels for required fields are suffixed by a red asterisk.
+<ExampleCodeBlock code={Grow} />
 
-<ExampleCodeBlock code={Required} />
+### Menu Height
 
-### Error States
+`Select.Card` has a default maximum height of `300px` to restrict the height of the dropdown menu.
+Set its `maxHeight` prop to override this value.
 
-Set the `error` prop of the wrapping Form Field to `FormField.ErrorType.Alert` or
-`FormField.ErrorType.Error` to set the Select to the Alert or Error state, respectively. You will
-also need to set the `hintId` and `hintText` props on the Form Field to meet accessibility
-standards.
+<ExampleCodeBlock code={MenuHeight} />
```

Comment:
I renamed the `Scrollable` section and example to `Menu Height` to better reflect its content related to `maxHeight`.